ts/features/bonus/cgn/screens/merchants/CgnMerchantsCategoriesSelectionScreen.tsx

```diff
@@ -88,9 +91,22 @@ const CgnTabBar = ({ state, navigation }: MaterialTopTabBarProps) => {
   );
 };
 const CgnMerchantsCategoriesSelectionScreen = () => {
+  const { navigate } =
+    useNavigation<
```

Comment:
Consider using `useIONavigation` when using navigation that is already typed and you no longer need to force the type here.